be/src/olap/rowset/segment_v2/column_writer.cpp

```diff
@@ -250,22 +249,24 @@ Status ColumnWriter::write_data() {
         _page_builder->get_dictionary_page(&dict_page);
         std::vector<Slice> origin_data;
         origin_data.push_back(dict_page.slice());
-        RETURN_IF_ERROR(_write_physical_page(&origin_data, &_dict_page_pp));
+        RETURN_IF_ERROR(_write_slices(&origin_data));
     }
     return Status::OK();
 }
 
 Status ColumnWriter::write_ordinal_index() {
     Slice data = _ordinal_index_builder->finish();
     std::vector<Slice> slices{data};
-    return _write_physical_page(&slices, &_ordinal_index_pp);
+    auto st = _write_slices(&slices);
+    return st;
 }
 
 Status ColumnWriter::write_zone_map() {
     if (_opts.need_zone_map) {
         OwnedSlice data = _column_zone_map_builder->finish();
-        std::vector<Slice> slices{data.slice()};
-        return _write_physical_page(&slices, &_zone_map_pp);
+        std::vector<Slice> slices;
+        slices.push_back(data.slice());
```

Comment:
why not use constructor without push_back

Reply:
done